src/assets/languages/english.json

```diff
@@ -67,6 +67,29 @@
         "registrationFail": "Registration unsuccesful"
       }
     },
+    "userUpdate": {
+      "messages": {
+        "heading": "Update your profile!",
+        "buttonMessage": "UPDATE",
+        "hoverProfile": "Click to upload",
+        "firstUpload": "You don't seem to have. Upload one :)",
```

Comment:
I don't think this line is required.

Reply:
@shreyans29 @paavininanda Is this fine? I feel it sounds very informal. Please add your comments.

Reply:
Yea.. this can be removed.